sw/device/silicon_creator/mask_rom/e2e_tests/mask_rom_e2e_testplan.hjson

```diff
@@ -0,0 +1,360 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "mask_rom_e2e"
+
+  testpoints: [
+    // Bootup without ROM_EXT
+    {
+      name: mask_rom_e2e_bootup_no_rom_ext
+      desc: '''Verify device fails to boot with no ROM_EXT.
+            
+            - Test runner loads the Mask ROM and no ROM_EXT.
+            - Attempt to boot.
+            - Verify that the chip indicates the correct boot fault.
+            '''
+      target: ["verilator", "dv_sim", "fpga", "silicon"]
```

Comment:
Should be `tags`. s/dv_sim/dv. 'dv' already implies 'simulation'.

Reply:
We will not (initially) because the host side component will be written in Rust, and use opentitanlib. There are no DPI hooks on the DV testbench for the various peripherals (e.g., SPI, UART, GPIO, etc.) like there are for Verilator. So we are targeting FPGA for almost all of these tests. Only tests that we absolutely cannot write in Rust,  e.g., tests that require forcing an alert, will the host side component be written in SV and run in DV. Additionally self-contained / self-checking tests can be run in DV.

Reply:
Unless I overlooked said DPI hooks, then there won't be an issue.

Reply:
Switched `target` to `tags` and `dv_sim` to `dv`. Also added `milestone: V2`